boards/esp32s3-pros3/Kconfig

```diff
@@ -25,6 +25,7 @@ config BOARD_ESP32S3_PROS3
     select HAS_TINYUSB_DEVICE
     # Only this board has a requirement to use USB_BOARD_RESET with STDIO_USB_SERIAL_JTAG
     select MODULE_USB_BOARD_RESET if MODULE_STDIO_USB_SERIAL_JTAG
+    select MODULE_USBUS_CDC_ACM if MODULE_USBUS
```

Comment:
Wouldn't it be more accurate to select this if the corresponding stdio is set?

Reply:
It cannot set the `MODULE_STDIO_CDC_ACM` without `MODULE_USBUS_CDC_ACM` which is why the `MODULE_STDIO_USB_SERIAL_JTAG` gets chosen.

Normally there would be a `FORCE_USB_STDIO`  which would select the `REQUIRES_USB_STDIO` that would automatically bring it in with `MODULE_USBUS`,

Maybe instead we do:
```suggestion
    select REQUIRES_USB_STDIO if MODULE_USBUS || MODULE_TINYUSB_DEVICE
```

I just ran, it seems like it would pass:

```
$ ./dist/tools/compile_test/compile_like_murdock.py -j 8 -a examples/hello-world/ tests/sys/fido2_ctap/ tests/sys/usbus_cdc_acm_stdio/ tests/sys/usbus_hid/ tests/sys/usbus_board_reset/ tests/sys/usbus_cdc_ecm/ tests/sys/usbus_msc/ tests/pkg/tinyusb_cdc_acm_stdio/ tests/pkg/tinyusb_netdev/ tests/pkg/tinyusb_cdc_msc/  -b esp32s3-pros3 -v
examples/hello-world/          esp32s3-pros3                  PASS
tests/sys/fido2_ctap/          esp32s3-pros3                  PASS
tests/sys/usbus_cdc_acm_stdio/ esp32s3-pros3                  PASS
tests/sys/usbus_hid/           esp32s3-pros3                  PASS
tests/sys/usbus_board_reset/   esp32s3-pros3                  PASS
tests/sys/usbus_cdc_ecm/       esp32s3-pros3                  PASS
tests/sys/usbus_msc/           esp32s3-pros3                  PASS
tests/pkg/tinyusb_cdc_acm_stdio/ esp32s3-pros3                  PASS
tests/pkg/tinyusb_netdev/      esp32s3-pros3                  PASS
tests/pkg/tinyusb_cdc_msc/     esp32s3-pros3                  PASS
This took 0:07:24.991030...
You could have 17799.641200000002 meters driven in a Tesla
```